flixel/FlxCamera.hx

```diff
@@ -555,13 +568,23 @@ class FlxCamera extends FlxBasic
 		{
 			_helperMatrix.identity();
 			_helperMatrix.translate(destPoint.x + frame.offset.x, destPoint.y + frame.offset.y);
-			var isColored:Bool = (cr != 1.0) || (cg != 1.0) || (cb != 1.0) || (ca != 1.0);
```

Comment:
It seems a little strange to me that alpha would be included in the `isColored` check, when it ends up being used for the `TILE_RGB` flag, which doesn't include alpha... Instead, alpha is always added to the draw data array. The original check might already have been wrong?